metricbeat/module/munin/_meta/fields.yml

```diff
@@ -4,8 +4,10 @@
     Munin node metrics exporter
   release: beta
   fields:
-    - name: munin
-      type: group
+    - name: munin.*
+      type: object
+      object_type: double
+      object_type_mapping_type: 'long'
```

Comment:
why long here? I would perhaps expect:
`object_type_mapping_type: "*"`

Reply:
I needed objects to keep being objects, I have added two patterns now, one for the level of groups and another one for the metrics.